Omni_vehicle:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  left_controller:
    type: effort_controllers/JointVelocityController
    joint: rim_left_joint
    pid: {p: 0.002,i: 0,d: 0}
  right_controller:
    type: effort_controllers/JointVelocityController
    joint: rim_right_joint
    pid: {p: 0.002,i: 0,d: 0}
  back_controller:
    type: effort_controllers/JointVelocityController
    joint: rim_back_joint
    pid: {p: 0.002,i: 0,d: 0}
